samples/js/counter/.eslintrc.js

```diff
@@ -0,0 +1,13 @@
+// Copyright 2016 Attic Labs, Inc. All rights reserved.
+// Licensed under the Apache License, version 2.0:
+// http://www.apache.org/licenses/LICENSE-2.0
+
+module.exports = {
+  rules: {
+    // Allow console
+    'no-console': 0,
+    // Used to distinguish between user errors and exceptions.
+    'no-throw-literal': 0,
+  },
+  extends: 'noms',
+};
```

Comment:
We can go back to symlinking. I initially thought it was silly to symlink `.babelrc` and `.eslintrc.js` since they are pretty much just a single line (like a symlink 🙂) anyway.